### PR TITLE
Refactor workspace tab bar

### PR DESCRIPTION
**Pull request by dmohs**
Prep for adding notifications component to right-hand side. Part of https://broadinstitute.atlassian.net/browse/GAWB-1551

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [x] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Assign to submitter** to finalize
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Squash commits and merge to develop
- [x] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by dmohs** [comment]
screen shot 2017-03-24 at 3 37 02 pm
screen shot 2017-03-24 at 3 35 55 pm
screen shot 2017-03-24 at 3 36 18 pm
screen shot 2017-03-24 at 3 36 41 pm

File: src/cljs/main/broadfcui/page/workspace/details.cljs [hunk 1]
```diff
@@ -18,59 +18,39 @@
...
+     (when (= false (:bucket-access? props))
+       [:div {:style {:paddingTop 2}}
+        [:div {:style {:backgroundColor "#efdcd7"
+                       :fontSize "small"
+                       :padding "4px 0"
+                       :textAlign "center"}}
+         (cond (= 404 (:bucket-status-code props))
+           (str "The Google bucket associated with this workspace"
+                " does not exist. Please contact help@firecloud.org.")
+           :else (str "The Google bucket associated with this workspace is currently"
```
Comment (zarsky-broad) [inline src/cljs/main/broadfcui/page/workspace/details.cljs:43]: Is this a place where clojure's built in multiline string concat would be sufficient?

File: src/cljs/main/broadfcui/page/workspace/details.cljs [hunk 2]
```diff
@@ -172,6 +186,8 @@
          (react/call :refresh-workspace this))))
    :component-did-update
    (fn [{:keys [prev-state state refs]}]
-     ;; when switching to the analysis tab, grab the ref to it so we can access the track picker button
-     (when (not= (:analysis-tab prev-state) (@refs ANALYSIS))
-       (swap! state assoc :analysis-tab (@refs ANALYSIS))))})
+     (if-let [analysis-tab (@refs ANALYSIS)]
```
Comment (zarsky-broad) [inline src/cljs/main/broadfcui/page/workspace/details.cljs]: Honestly, I still don't like having this button here, but if we keep it, it's too tall. It should be the same height as the import attributes button in summary.

**Review by zarsky-broad — APPROVED** [review]
Looking good. Couple things, minor.
(and thanks for the screenshots)

File: src/cljs/main/broadfcui/page/workspace/details.cljs (hunk 1 above)
Comment (dmohs) [reply src/cljs/main/broadfcui/page/workspace/details.cljs:43]: It appears not. The compiler sees that as two separate forms.

**Comment by dmohs** [comment]
screen shot 2017-03-27 at 2 23 29 pm

**Pull request merged**